keps/sig-node/1558-streaming-proxy-redirects/kep.yaml

```diff
@@ -14,9 +14,10 @@ approvers:
   - "@Random-Liu"
   - "@derekwaynecarr"
 creation-date: 2019-12-05
+last-updated: 2021-05-11
 status: implementable
 replaces:
   - "https://docs.google.com/document/d/1OE_QoInPlVCK9rMAx9aybRmgFiVjHpJCHI9LrfdNM_s/edit#heading=h.4yfjiw58o8d3"
 
-latest-milestone: "0.0"
-stage: "alpha"
+latest-milestone: "1.22"
+stage: "stable"
```

Comment:
This should not say stable, as it's a deprecation. I think Sergey set it to `beta (deprecated)` in another PR.